src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -207,6 +208,20 @@ public function testParsesTags()
         }
     }
 
+    public function testParseTagsWithoutName()
+    {
+        try {
+            $container = new ContainerBuilder();
+            $loader = new XmlFileLoader($container, new FileLocator(self::$fixturesPath.'/xml'));
+            $loader->load('invalid_tag.xml');
+
+            $this->fail('The XmlFileLoader is expected to throw an exception when the name attribute is missing or a tag.');
+        } catch (InvalidArgumentException $e) {
+            $this->assertInstanceOf('\InvalidArgumentException', $e->getPrevious());
+            $this->assertRegExp("/The attribute 'name' is required but missing/", $e->getPrevious()->getMessage());
```

Comment:
Well, your change is precisely about making the XSD validation fail when the attribute is not there. What I mean is that we should probably avoid relying on the exact wording of the error message in the testsuite (new versions of PHP may alter the message)